XClass/index.js

```diff
@@ -0,0 +1,51 @@
+const Plugin = module.parent.require('../Structures/Plugin');
+
+class XClass extends Plugin {
+    constructor(...args) {

```

Comment:
Instead of `constructor(...args) ` use `load()`

Reply:
using load makes more semantic sense but either are fine. it should be noted that load will perform faster than the constructor as it doesn't have to use the rest operator